QStore4S/src/main/java/edu/asu/qstore4s/repository/AppellationEventRepository.java

```diff
@@ -9,23 +9,28 @@
 import edu.asu.qstore4s.domain.events.impl.AppellationEvent;
 import edu.asu.qstore4s.domain.events.impl.RelationEvent;
 
-public interface AppellationEventRepository extends GraphRepository<AppellationEvent> {
-
-	AppellationEvent findById(String id);
-
-	@Override
-	@Transactional
-	public <U extends AppellationEvent> U save(U arg0);
-	
-	/**
-	 * The method find parent relation event from appellation event
-	 * @param appellationEvent
-	 * @return
-	 */
-
-	@Query("start result = node({0}) " +
-			"match (result)<-[hasSubject]-(relation)<-[hasRelation]-(relationEvent) " +
-						"return relationEvent limit 10")
-	List<RelationEvent> getRelationfromId(AppellationEvent appellationEvent);
-	
+public interface AppellationEventRepository extends
+        GraphRepository<AppellationEvent> {
+
+    public AppellationEvent findById(String id);
+
+    @Override
+    @Transactional
+    public <U extends AppellationEvent> U save(U arg0);
+
+    /**
+     * The method find parent relation event from appellation event

```

Comment:
This is not 100% correct. the method returns all RelationEvents that reference the given AppellationEvent.